pkg/metrics/datacoord_metrics.go

```diff
@@ -149,6 +149,26 @@ var (
 			Buckets:   buckets,
 		}, []string{segmentFileTypeLabelName})
 
+	/* garbage collector related metrics */
+
+	// GarbageCollectorListLatency metrics for gc scan storage files.
+	GarbageCollectorListLatency = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: milvusNamespace,
+			Subsystem: typeutil.DataCoordRole,
+			Name:      "gc_list_latency",
+			Help:      "latency of list objects in storage while garbage collecting (in seconds)",
+			Buckets:   []float64{0.001, 0.1, 0.5, 1, 5, 10, 20, 50, 100, 250, 500, 1000, 3600, 5000, 10000}, // unit seconds
```

Comment:
We have a common aggreement that histogram buckets in milliseconds unit.

Reply:
this common bucket is too small for gc duration